src/python/era_5g_object_detection_standalone/era_5g_object_detection_standalone/interface.py

```diff
@@ -121,14 +125,85 @@ def post_image():
     for file in request.files.to_dict(flat=False)['files']:
         # print(file)
         nparr = np.fromstring(file.read(), np.uint8)
-        # decode image
-        img = cv2.imdecode(nparr, cv2.IMREAD_COLOR)
 
         # store the image to the appropriate task
-        task.store_image({"sid": sid, "websocket_id": task.websocket_id, "timestamp": timestamps[index]}, img)
+        # the image is not decoded here to make the callback as fast as possible
+        task.store_image({"sid": sid, 
+                          "websocket_id": task.websocket_id, 
+                          "timestamp": timestamps[index], 
+                          "decoded": False}, 
+                         nparr)
         index += 1
     return Response(status=204)
 
+@socketio.on('connect', namespace='/data')
+def connect_ws(auth):
```

Comment:
It would probably like to unify the names of all functions. E.g. naming "connect" and "connect_ws" doesn't make sense when they differ by namespace "/results" and "/data". Websocket connection are both functions.

Reply:
Agree

Reply:
done